src/site/_includes/js/test/eligibility-programs.test.js

```diff
@@ -504,6 +504,34 @@ describe('Program eligibility', () => {
     });
   }
 
+  function testDischargeStatus(setupFn, resultFn, testCases) {
+    describe.each(testCases)('Discharge status "$dischargeStatus"', (
+      {dischargeStatus, expectedFlag, expectedEligible}) => {
+      beforeEach(() => {
+        setupFn(input);
+      });
+
+      if (expectedEligible !== undefined) {
```

Comment:
is this leftover from some tinkering?  I'm not sure `expectedEligible` should ever be undefined in a test case.